### ISSUE TITLE
trying to download Git to Mac

### ISSUE THREAD

**Issue by grayhawk2003** [timcharper/git_osx_installer#167]
I'm trying to download through Source Forge on my Mac and keep getting "macOS cannot verify that this app is free from malware".  Also tried through Home Brew but I wasn't able to successfully pate the code into my laptops "Terminal".  Any other options or tips on how to download this onto my MacBook?  I'm not very tech savvy and just started a coding bootcamp, so PLEASE speak in layman's terms.  many thanks.  Meg

**Comment by cazmalone** [comment]
Hi Meg -- I am in the same position as you. I found a solution in another post. Good luck, CM
Seb-GJ commented on 28 Jul
If you have a Mac you need to go to your system preferences, open de privacy & security panel, in the window general there you need to allow the system to download programs from every site

**Comment by JPow** [comment]
If you're in Sierra and this option isn't available you can right click on the app and select open and I found that works.

**Comment by agnessinno** [comment]
> Hi Meg -- I am in the same position as you. I found a solution in another post. Good luck, CM
> Seb-GJ commented on 28 Jul
> If you have a Mac you need to go to your system preferences, open de privacy & security panel, in the window general there you need to allow the system to download programs from every site

I had the same problem so thank you @cazmalone! The only difference was that my Mac indicated exactly the website with link to be open ( https://sourceforge.net/projects/git-osx-installer/) 
so i didn't have to mark "download from every site".
